terraform/README.md

```diff
@@ -118,7 +151,8 @@ Next, you should activate it. This step is platform dependent:
 
 ### 2. Install requirements
 ```
-pip3 install -r requirements.txt
+pip install pyserial cryptography click future pyelftools setuptools
```

Comment:
Add line that venv needs to be activated

Reply:
Requirements installation follows the activation of virtual environments - there is information about it two lines above